public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -252,12 +244,12 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
   });
 }
 
-export function createSceneVariableFromVariableModel(variable: VariableModel): SceneVariable {
+export function createSceneVariableFromVariableModel(variable: TypedVariableModel): SceneVariable {
```

Comment:
@kaydelaney I do not think this change is correct, VariableModel here is the schema model imported from schema package,

Reply:
but it looks like we where not using the schema types here anyway before, argh..